app/src/test/kotlin/co/netguru/baby/monitor/client/feature/babycrynotification/NotifyBabyEventUseCaseTest.kt

```diff
@@ -0,0 +1,60 @@
+package co.netguru.baby.monitor.client.feature.babycrynotification
+
+import co.netguru.baby.monitor.RxSchedulersOverrideRule
+import co.netguru.baby.monitor.client.feature.babynotification.NotifyBabyEventUseCase
+import co.netguru.baby.monitor.client.feature.firebasenotification.FirebaseNotificationSender
+import co.netguru.baby.monitor.client.feature.firebasenotification.NotificationType
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.doReturn
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import org.junit.Rule
+import org.junit.Test
+
+class NotifyBabyEventUseCaseTest {
+
+    @get:Rule
+    val schedulersRule = RxSchedulersOverrideRule()
+
+    private val notificationSender: FirebaseNotificationSender = mock()
+    private val text = "text"
+    private val notificationTexts = mock<Map<String, String>> {
+        on { get(any<String>()) }.doReturn(text)
+    }
+
+    private val notifyBabyEventUseCase =
+        NotifyBabyEventUseCase(
+            notificationSender,
+            notificationTexts
+        )
+
+    @Test
+    fun `should send crying notification on notifyBabyCrying`() {
+        notifyBabyEventUseCase
+            .babyEvents()
+            .subscribe()
```

Comment:
You can assert result with `test()` method.

Reply:
There is nothing to assert here. Subscription lives as long as it isn't disposed. On complete isn't called.